catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopProxyPlugin.java

```diff
@@ -82,7 +82,7 @@ public Object doAs(
 
   @Override
   public void bindCatalogOperation(CatalogOperations ops) {
-    this.ops = ((HadoopCatalogOperations) ops);
+    this.ops = ((SecureHadoopCatalogOperations) ops);
```

Comment:
Should the plugin be null?

Reply:
`HadoopProxyPlugin` was not used anymore, so it does not matter here.